index.d.ts

```diff
@@ -0,0 +1,40 @@
+import * as Promise from 'bluebird';
+import TypedError = require('typed-error');
+
+declare module 'bluebird-lru-cache' {
+
+	export class NoSuchKeyError extends TypedError {
+		public constructor(
+			public key: string,
+		) {
+			super();
+		}
+	}
+
+	export interface Options {
+		max: number;
+		maxAge: number;
+		length: (n: any) => number;
+		dispose: (key: any, value: any) => any;
+		stale: boolean;
+		noreject: boolean;
+		fetchFn: (key: any) => any;
+	}
+
+	export default class BluebirdLRU {
+
+		public constructor(options: Options);
+
+		public set(key: any, value: any, max?: number): Promise<boolean>;
```

Comment:
Why `key` is typed as `any` everywhere, but `NoSuchKeyError` declares it as a `string`? I'd expect it to be either `any` or `string` everywhere.